Data/src/SQLChannel.cpp

```diff
@@ -301,7 +303,7 @@ void SQLChannel::run()
 		Thread::sleep(sleepTime);
```

Comment:
Would it be possible to use conditions instead of sleep?

Reply:
I used event with interruptible wait (to make sure timed flush occurs), good suggestion